core/app/services/uffizzi_core/compose_file/parsers/services/volumes_parser_service.rb

```diff
@@ -70,18 +74,15 @@ def build_volume_attributes(source_path, target_path, read_only, params = {})
     end
 
     def build_volume_type(source_path, target_path)
-      if path?(source_path) && path?(target_path)
-        raise UffizziCore::ComposeFile::ParseError, I18n.t('compose.volume_type_not_supported', type: HOST_VOLUME_TYPE)
-      end
-
+      return HOST_VOLUME_TYPE if path?(source_path) && path?(target_path)
       return ANONYMOUS_VOLUME_TYPE if path?(source_path) && target_path.blank?
       return NAMED_VOLUME_TYPE if source_path.present? && !path?(source_path) && path?(target_path)
 
       raise UffizziCore::ComposeFile::ParseError, I18n.t('compose.volume_path_is_invalid', path: [source_path, target_path].join(':'))
     end
 
     def path?(path)
-      /^(\/|\.\/|~\/)/.match?(path)
+      /^(\/|\.\/|\.\.\/)/.match?(path) # volume path should start with / or ./ or ../
```

Comment:
relative?

Reply:
Path can start with `/` and this is not relative